### PR TITLE
Update OllamaEmbeddings.py

### PR DESCRIPTION
**Pull request by yamonkjd**

### CODE REVIEW

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
Hey @yamonkjd ! 

This looks great. 

Can you run `make lint` just to make sure all the lint issues are resolved? Once those are fixed I'll merge it even if the tests are failling.

**Comment by yamonkjd** [comment]
@ogabrielluiz 


Yes, I'll check and leave additional feedback.

**Pull request closed without merging**
